src/content_delegate.js

```diff
@@ -144,65 +144,72 @@ ContentDelegate.prototype.findAndStorePacerDocIds = function() {
         debug(3, `Y doc ${pacer_doc_id} to ${goDLS.de_caseid}`);
       } else if (page_pacer_case_id) {
         docsToCases[pacer_doc_id] = page_pacer_case_id;
-        debug(3,`X doc ${pacer_doc_id} to ${page_pacer_case_id}`);
+        debug(3, `X doc ${pacer_doc_id} to ${page_pacer_case_id}`);
       }
     }
   }
-  chrome.storage.local.set(docsToCases, function(){
+
+  // identify current doc Number with simple key in chrome storage
+  const urlMatch = PACER.getDocumentIdFromUrl(document.URL);
+  if (!!urlMatch) {
+    saveItemToStorage({docId: urlMatch});
+  }
```

Comment:
I might have missed it, but does this get used somewhere? Sorry, I don't quite grok why you're doing this here.

Reply:
Yes, the uploadPDF function pulls the docID from storage

Reply:
Looks like uploadPDF gets the pacer_doc_id passed as a parameter, and the uploadZip function gets it from storage. Why not pass it through for the zip function like we do for the PDF function. It looks like in content_delegate.js you should be able to just do `this.pacer_doc_id`? Maybe?

Reply:
There is some peculiarity in the flow where the this.pacer_doc_id isn’t set. I think the doc id is set two pages before the download function. So you’d have to set the docID and then persist it for further down the line. Don’t know how they did it before but it seems cleaner to put it into the global store and retrieve it from there.

Reply:
OK, got it. Feels like there should be a better way to do this that doesn't use globally available persistence. I'm concerned about two things with this:

1. It's different than how we do PDFs, so it's two things to put in our heads.
2. It's not tied to the specific tab, so it could present race conditions even if a user is slow.

Imagine:

1. Go to a docket, choose a zip to download, stop at the receipt page.
2. New tab, go to a docket, choose a different zip to download, stop at the receipt page.
3. Go to the first tab and download it. 

That spells trouble, right? Could we use the tab `id` to tie the nonce to a specific tab: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/Tab